docs/dac/getting-started.md

```diff
@@ -7,49 +7,70 @@
 
-- `percli`, the [CLI of Perses](../cli.md).
-- `cue`, the [CLI of Cuelang](https://cuelang.org/).
+- `percli`, the [CLI of Perses](../cli.md). ⚠️ The version should be at least `v0.51.0`.
+- `cue`, the [CLI of Cuelang](https://cuelang.org/). ⚠️ The version should be at least `v0.12.0`.
 
 ### Repository setup
 
 Create a new folder that will become your DaC repository, then follow the steps below:
 
 #### 1. Initialize the CUE module
 
-```
+You first have to initialize a CUE module in order to be able to import the SDK dependency afterwards:
+
+```bash
 cue mod init <module name>
 ```
 
 See the [CUE documentation](https://cuelang.org/docs/concept/modules-packages-instances/) for more information about this step.
 
-#### 2. Retrieve the CUE sources from Perses
+#### 2. Install the Perses SDK
 
-Ideally we should rely on a native dependency management here, but since it's not yet available for CUE as already mentioned, we provide in the meantime a dedicated CLI command `dac setup` in order to add the CUE sources from Perses as external dependencies to your repo:
+First, log in to the Central Registry of CUE. This will allow to install the Perses SDK afterwards (as it consists of importing the `github.com/perses/perses/cue` CUE module). ⚠️ For now this login step is only possible using a Github account.
 
+```bash
+cue login
 ```
-percli dac setup --version 0.47.1 # any version you'd like above v0.44.0
+
+Now you can run the setup command that percli provides in order to install the SDK:
+
+```bash
+percli dac setup
 ```
 
-You can omit the version flag if you are connected to a Perses server (it will retrieve its version). Otherwise, unless you have a specific case, better to pass the latest version available.
+Check the helper of that command to see the different flags available (e.g to define the SDK version to install).
 
 ### Develop dashboards
 
-You are now fully ready to start developping dashboards as code!
+You are now fully ready to start developping dashboards as code with CUE!
 
 It's first strongly recommended to ramp up on CUE if you are not familiar with this technology. For this have a look at:
 
 - The [official website](https://cuelang.org/) of Cuelang.
 - [Cuetorials](https://cuetorials.com/), a 3rd party source of information that is a very good complement.
 
-You should then have a look at the [CUE SDK documentation](../dac/cue/README.md) to better understand how to use the framework.
+You should then have a look at the [CUE SDK documentation](../dac/cue/README.md) to better understand how to use it.
 
 You can also check an example of DaC usage [here](https://github.com/perses/perses/blob/main/internal/test/dac/input.cue).
 
+For each [plugin](../concepts/plugins.md) you would like to use in your DaC, it is strongly recommended to import its module so that you benefit from its schema validation locally. Check the list of available plugins & their corresponding module name at https://github.com/perses/plugins.
```

Comment:
_"Check the list of available plugins & their corresponding module name at https://github.com/perses/plugins"_ to be improved later when plugin docs will be also be published to the documentation website